src/templates/Footer/footer.module.scss

```diff
@@ -26,6 +26,8 @@
 
   &-phone {
     @include typography-6(false);
+    color: inherit;
+    font-weight: 700;
```

Comment:
@petrgazarov This font-weight should be set with our mixin. We don't explicitly write font-weights anymore.